assets/styl/_modules/_required.styl

```diff
@@ -0,0 +1,10 @@
+// Requided Marker
+//
+// Markup: required.twig
+//
+// Styleguide Components.requided
```

Comment:
This can be discussed, but I think namings and position in the do should be related to Forms, for example:
- for files `_form-required.styl` and `form-required.twig``
- in the doc: `Components.Forms.Required`